test/catkin_package_tests/p1/foo/package.xml

```diff
@@ -14,9 +14,6 @@
   <author>John Doe</author>
   <author email="jane.doe@example.com">Jane Doe</author>
 
-  <build_depend>catkin</build_depend>
-  <build_depend version_gte="1.1" version_lt="2.0">genmsg</build_depend>
-
```

Comment:
Why were these dependencies removed in 32e65807169cdd3e4435f1311db93c601b46fdfd?

Reply:
According to the commit message, unit test fails as the dependency `genmsg` doesn't get installed for `rospkg` itself nor for the unit test. If there's a non-hacky way to install the test dependency I'm happy to add that as well as getting these lines back in.